package.json

```diff
@@ -47,7 +47,7 @@
     "eslint-plugin-jest": "23.0.4",
     "eslint-plugin-sonarjs": "0.5.0",
     "jest": "24.9.0",
-    "mysql2": "1.6.5",
+    "mysql2": "2.1.0",
```

Comment:
According to semver, this upgrade should contain breaking changes.
If so, did you took the time to check the potential compatibility issues?

Reply:
- I read the changelog: https://github.com/sidorares/node-mysql2/blob/master/Changelog.md
- I updated the tests to test more versions: https://github.com/ForestAdmin/forest-express-sequelize/pull/373/files#diff-0fd0e07cf6d02bf7cf00f18cebb8e6eaR36
- I installed a new project with lumber for MySQL 8 and tested it works in our interface.

Maybe it's not enough, even if the changelog seems to say there is no "real" breaking change. Feel free to suggest some other tests.